blog/2024-09-07-shadow-bucket-thumbnails/index.md

```diff
@@ -0,0 +1,236 @@
+---
+slug: shadow-bucket-thumbnails
+title: Using Shadow Buckets for Fun and Thumbnails
+description: >
+  Exploring Tigris' shadow bucket feature for fun and thumbnails. In the blog
+  post the feature will be used in an less conventional fashion. An additional
+  elixir service will generate thumbnails, where source files as well as
+  generated thumbnails are stored on Tigris, while making use of the included
+  CDN features of Tigris.
+keywords: [object storage, blob storage, s3, thumbnail, images, cloudfront, cdn]
+authors: [bmi]
```

Comment:
Add the following so that the social image is picked up correctly in link previews:

```
image: ./a-pair-of-scissors-cutting-a-photograph.jpg
```